packages/trackers-preview/src/content_scripts/index.js

```diff
@@ -118,15 +118,6 @@ export default function setupTrackersPreview(popupUrl) {
           return;
         }
 
-        document.addEventListener('click', (event) => {
```

Comment:
This event listener was only to hide the popup when clicking on the result item link. I think it's not important (I am not sure why we added it here), and it can't be in that form with reloading the setup for the endless scroll case (it would add multiple events on the document).

Reply:
Unfortunately, I cannot try it out, since I'm currently unable to get any Tracker Preview popup at all. (Not tested with this PR, but in general. Don't know why.)

Maybe the feature is to have a convenient way to close the popup by clicking on anything outside the popup itself to close it (not only on the X). I could see that this adds value.

Reply:
It works only for links, so when you navigate out from the page.